### ISSUE TITLE
Video not autoplaying on iOS

### ISSUE THREAD

**Issue by ian** [cloudinary/cloudinary-react#150]
Code sample: 

```
<Video
  cloudName={process.env.REACT_APP_CLOUDINARY_CLOUD_NAME}
  publicId={video}
  autoPlay
  playsinline
  loop
  muted
/>
```

iOS is particularly difficult to autoplay video.  I wonder if it has to do with the `playsinline` field.  <Video> doesn't seem to passthrough the playsinline video.  

More details here: https://webkit.org/blog/6784/new-video-policies-for-ios/ 

Platforms:
iOS, works fine on chrome/safari

**Comment by michalkcloudinay** [comment]
HI @ian, This should work in IOS. Can you specify in what IOS version and browser you had this issue?

**Comment by maoznir** [comment]
Hi @ian
Due to how React handles element attributes, 'playsinline' should be 'playsInline'.
Please let us know is this solves it for you

**Comment by el-noppo** [comment]
This would work, up until 7 days ago, when you decided to snake case any options passed to the Video tag.

**Comment by maoznir** [comment]
@el-noppo Sorry about that and thanks for pointing this out, it's now fixed in 1.3.4
